woocommerce/woocommerce-ios
# [Analytics Hub] Add line chart with placeholder data to AnalyticsReportCard

Part of #8201 
⚠️ Depends on #8230 ⚠️ 

## Description

Adds the `AnalyticsLineChart` view to the `AnalyticsReportCard`:

* For now, the chart is displayed with placeholder data. This will be replaced with real data (interval data from the order data API response) in a following PR.
* The chart uses the same color as the `DeltaTag` background.

## Changes

* Adds placeholder chart data to `AnalyticsHubViewModel`.
* Adds a leading and trailing chart using `AnalyticsLineChart` view to `AnalyticsReportCard` and its view model.

## Testing

1. Launch the app.
2. Tap the "See More" button from the main dashboard
3. See the Analytics view loads with the sample charts on the Revenue and Orders cards.

## Screenshots

<img src="https://user-images.githubusercontent.com/8658164/204289129-47bb31e7-a64b-415e-89c8-50d0eb56fbcc.png" width="300px">

## Submitter Checklist

Update release notes:
